docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3"
+services:
+    mldonkey:
+        stdin_open: true
+        tty: true
+        environment:
+            - MLDONKEY_ADMIN_PASSWORD=$ADMIN_PWD
+            - MLDONKEY_UID=1000
+            - MLDONKEY_GID=1000
+            - TZ=$TZ
+        ports:
+            - 4000:4000
+            - 4001:4001
+            - 4002:4002
+            - 4080:4080
+            - 4081:4081
+            - 8844:8844
+            - 8848:8848/udp
+            - 17043:17043
+            - 17043:17043/udp
+            - 3820:3820
+            - 3820:3820/udp
```

Comment:
Instead of listing only these ports, you may set the ports like in the other example.